.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+max-line-length = 100
+output-file = flake8_log.txt
+tee = True
+ignore = E722,E203,W503,T001
```

Comment:
Are there specific reasons for these?

E722: I generally agree with this rule as 'catch all' statements can result in hard to find bugs
E203 and W503: Should be pretty easy to setup auto formatting to have the spacing consistent. Do we need this ignore?
T001: Generally a good reminder to log instead of print and we can always add ignores on a one-off basis

Reply:
Thanks for catching this. It was actually pasted from another project. I'll remove them. I agree that all of these should just be removed to ensure optimal code styling. What do you think of `max-line-length` of 100? I generally like this since the default line length of `79` can get very cumbersome, especially when it comes to docstrings and comments.

Reply:
Yeah, I agree 79 is a bit short